src/database/db_update.js

```diff
@@ -0,0 +1,15 @@
+const connect = require('./db_connect');
+
+const updateLikes = (likeNum, memoryId) => {
```

Comment:
Okay, I do not think you understood me, this function will no longer work. You cannot return from an async function, it does not work!

Before you had
```js
const updateLikes = (arg, cb) => {
  query('SOME QUERY', [], (err, res) => {
    if (err) {
      cb(err);
      return;
    }
    cb(null, res)
  }
}
```

I was suggesting you changed it too

```js
const updateLikes = (arg, cb) => {
  query('SOME QUERY', [], cb)
}
```

As these two are identical